### PR TITLE
Add SNAP emergency allotment

### PR DESCRIPTION
**Pull request by MaxGhenis**

### CODE REVIEW

File: openfisca_us/parameters/usda/snap/emergency_allotment/minimum.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: Minimum amount of SNAP emergency allotment.
+
+CA:
+  2022-01-01: 95
+
+metadata:
+  period: month
+  reference:
+    - title: "California: COVID-19 Waivers & Flexibilities"
+      href: https://foodforpeople.org/covid-19-calfresh
```
Comment (MaxGhenis) [inline openfisca_us/parameters/usda/snap/emergency_allotment/minimum.yaml:10]: I can't find a legislative reference for this. It's probably here: https://www.cdss.ca.gov/inforesources/legislation-and-regulations

But two sites have mentioned the $95 minimum.

**Comment by MaxGhenis** [comment]
Pretty much all set here, just going to revise the notebook to show the normal rules in addition to the emergency rules, by setting `snap_emergency_allotment` to 0.

**Comment by MaxGhenis** [comment]
Ready for review, though I suspect the jupyterbook won't render. @nikhilwoodruff would you mind running/pushing the notebook for me?

**Review by nikhilwoodruff — APPROVED** [review]
Nice, thanks @MaxGhenis

**Pull request merged**